webrtc-stats.html

```diff
@@ -1569,8 +1570,20 @@ <h2>
                   Transport Independent Application Specific (TIAS) bitrate [[!RFC3890]].
                   Typically, the target bitrate is a configuration parameter provided to the
                   codec's encoder and does not count the size of the IP or other transport layers
-                  like TCP or UDP. It is measured in bits per second and the bitrate is calculated
-                  over a 1 second window.
+                  like TCP or UDP. It is measured in bits per second. This statistic gives the
+                  configured target bitrate at the time of the stats timestamp.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>totalTargetBytesEncoded</code></dfn> of type <span class=
+                "idlMemberType">unsigned long long</span>
+              </dt>
+              <dd>
+                <p>
+                  The total number of bytes the encoder would have produced if it exactly matched
+                  the configured target. In reality, the actual number of bytes may be higher or
+                  lower on a frame-by-frame basis. This value is updated each time a frame has
+                  been encoded by incrementing its value by target bytes for that frame.
```

Comment:
Suggested formulation: "This value is increased by the target frame size in bytes every time a frame has been encoded. The actual frame size may be bigger or smaller than this number."

I'm not sure the target frame size is constant. Do keyframes get a larger target size than non-keyframes?

Reply:
Target Size is for the frame to be encoded, right?

Would changing "has been encoded" --> "is to be encoded" mislead implementors into counting target bytes when no frame was generated?

Reply:
Yes, I think it would mislead people.
OTOH, if we drop a frame because we have overshot the target bitrate in the previous frame, then it makes sense to increase the totalEncodedBytesTarget, so perhaps "is to be encoded" is the right formulation.

Reply:
want to make sure a non-encoded frame should not be counted towards the byte.

Reply:
Clarify: only goes up when framesEncoded goes up.
Split up PR into new stat and clarifying old stat

Reply:
btw what is the difference between the bytesSent (payload only) and totalTargetBytesEncoded?

One difference that I foresee is that the two metrics are updated at different moments in time.